op-challenger/fault/agent.go

```diff
@@ -57,11 +56,13 @@ func (a *Agent) Act(ctx context.Context) error {
 // and returns true if the game resolves successfully.
 func (a *Agent) tryResolve(ctx context.Context) bool {
 	if a.responder.CanResolve(ctx) {
+		a.log.Info("Resolving game")
 		err := a.responder.Resolve(ctx)
 		if err != nil {
-			return true
+			a.log.Error("Failed to resolve the game", "err", err)
+			return false
 		}
-		a.log.Error("failed to resolve the game", "err", err)
+		return true
 	}
 	return false
```

Comment:
I think we can do something like

```suggestion
        if err == nil {
            return true
        }
        a.log.Error("Failed to resolve the game", "err", err)
    }
    return false
```

Reply:
Will open a new pr so we can get these merged.